src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -101,6 +101,14 @@ a#editUncommonSettingsButton:hover{
     cursor: hand;
 }
 
+.btn.btn-enable-mouse-events {
+    pointer-events: auto;
+}
+
+.btn.btn-enable-mouse-events:active {
```

Comment:
Just tested, seems it isn't all that necessary as I thought, using `!important` seems to work now, although it didn't seem to work previously.

Also changing the name as it is non-representative of the current behaviour, as it currently only allows for hover events. Previously it was intended to allow all events, but manually stop click events via JS.